manifests/plugin.pp

```diff
@@ -75,6 +81,7 @@
     $source      = undef,
     $proxy_host  = undef,
     $proxy_port  = undef,
+    $plugin_dir  = undef,
```

Comment:
We can pass the plugin_dir from the params class here so we get the right one depending the distro. This way the user doesn't have to specify it them selves.
Also the plugin dir is set via the main class and passed on internally setting different variables ( including config file )